integration/simple-cache-client.test.ts

```diff
@@ -375,3 +376,13 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
     ).toEqual(false);
   });
 });
+
+describe('Integration Tests for operations on sets datastructure', () => {
```

Comment:
I guess you need SetAdd to do more testing. Would you add a test for an invalid set name?

Reply:
Yea there'll be more test cases once I add the other operations.